packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -731,10 +728,7 @@ test.describe('data-sveltekit attributes', () => {
 			page.waitForLoadState('networkidle') // wait for preloading to finish
 		]);
 
-		expect(page).toHaveURL(
-			'/data-sveltekit/preload-data/offline/slow-navigation' ||
-				(browserName === 'chromium' && 'chrome-error://chromewebdata/')
```

Comment:
why did this change?

Reply:
ESLint reported always true condition. `'non-empty-string' || (whatever)` will always return `'non-empty-string'`, unless I'm missing something obvious.